src/lib.rs

```diff
@@ -47,11 +80,15 @@ pub fn parse_and_inline_modules(src_file: &Path) -> syn::File {
 #[derive(Debug)]
 pub struct InlinerBuilder {
     root: bool,
+    annotate_paths: bool,
```

Comment:
Given that this changes the output, do you think it's worth including as a field in `InliningResult`? I'm not strongly attached to the idea, so if it's not something you'd use feel free to leave it out.

Reply:
That's a pretty good idea :)